src/utils.rs

```diff
@@ -179,13 +179,19 @@ pub fn rearrange_columns_and_rewrite(filename: &str, sequence: Vec<String>) {
         .expect("Cannot read file")
         .with_delimiter(b'\t')
         .finish()
-        .unwrap();
+        .expect("Error reading CSV file");
 
     // Change the sequence of the columns of the TSV file
-    let mut df_reordered = df.select(sequence).unwrap();
+    let mut df_reordered = df.select(sequence).expect("Error selecting columns");
 
     // Use writer to write df_reordered into a TSV file.
-    let mut buf = File::create(filename).unwrap();
+    // let mut buf = File::create(filename).unwrap();
+    let mut buf = OpenOptions::new()
```

Comment:
I learned about OpenOptions from this, right now - if an error gets raised, will the output just be "Error reading CSV file" or will it also provide the actual error like [PermissionDenied](https://doc.rust-lang.org/std/io/enum.ErrorKind.html#variant.PermissionDenied) etc?

Reply:
As per this code it will just Show the message as you mentioned above. I'll edit the code such that it gives us more details 
```
.unwrap_or_else(|error| {
        panic!("Error opening file '{}': {}", filename, error);
    });
```

Reply:
Excellent